etna/analysis/plotters.py

```diff
@@ -116,51 +189,69 @@ def plot_forecast(
         else:
             plot_df = pd.DataFrame(columns=["timestamp", "target", "segment"])
 
-        segment_forecast_df = forecast_ts[:, segment, :][segment]
-
         if (train_ts is not None) and (n_train_samples != 0):
             ax[i].plot(plot_df.index.values, plot_df.target.values, label="train")
         if test_ts is not None:
             ax[i].plot(segment_test_df.index.values, segment_test_df.target.values, color="purple", label="test")
-        ax[i].plot(segment_forecast_df.index.values, segment_forecast_df.target.values, color="r", label="forecast")
-
-        if prediction_intervals and quantiles is not None:
-            alpha = np.linspace(0, 1, len(quantiles) // 2 + 2)[1:-1]
-            for quantile in range(len(quantiles) // 2):
-                values_low = segment_forecast_df["target_" + str(quantiles[quantile])].values
-                values_high = segment_forecast_df["target_" + str(quantiles[-quantile - 1])].values
-                if quantile == len(quantiles) // 2 - 1:
-                    ax[i].fill_between(
-                        segment_forecast_df.index.values,
-                        values_low,
-                        values_high,
-                        facecolor="g",
-                        alpha=alpha[quantile],
-                        label=f"{quantiles[quantile]}-{quantiles[-quantile-1]} prediction interval",
-                    )
-                else:
-                    values_next = segment_forecast_df["target_" + str(quantiles[quantile + 1])].values
-                    ax[i].fill_between(
+
+        # plot forecast plot for each of given forecasts
+        colors = plt.cm.Set2.colors
```

Comment:
i'm not sure it's a good idea to set color scheme in the method itself

we were going to set common scheme for all the plots, so this method will reset all the settings 🤔

Reply:
This line don't override anything, here I just take one particular scheme and use colors in it as a list. I also can take the default color scheme. I think that it can be reset from the outside if we will choose one particular version.

Should I do it?